src/Gui/Windows/Controls/TextViewModel.cs

```diff
@@ -99,10 +99,34 @@ public interface TextViewModel
     /// </summary>
     public class EmptyEditorModel : TextViewModel
     {
-        public object CurrentPosition { get { return this; } }
-        public object StartPosition { get { return this; } }
-        public object EndPosition{ get { return this; } }
-        public int LineCount { get { return 0; } }
+        public object CurrentPosition
+        {
+            get
+            {
+                return null;
+            }
```

Comment:
Is there a reason why these were changed to `null`? I'd like to avoid injecting invalid objects into the live object graph where possible.